package.json

```diff
@@ -262,6 +267,7 @@
     "prettier": "^1.19.1",
     "react-test-renderer": "^16.12.0",
     "rimraf": "^3.0.0",
+    "sass": "^1.53.0",
```

Comment:
Can we remove `node-sass` now?

Reply:
After `node-sass` removal I had some errors after cleaning the repo and `yarn`. I leave it as it is for now